tests/test_compute.py

```diff
@@ -428,4 +441,4 @@ def test_compute_environments(client):
     compute_envs_endpoint = BaseURLs.SERVICES_URL + "/computeEnvironments"
     response = client.get(compute_envs_endpoint)
 
-    assert "environments" in response.json
+    assert "id" in response.json[0]
```

Comment:
since we are using test compute_helpers, I think we can also assert some actual contents, not just the presence of the key in the response